src/main/resources/component-kernel.conf

```diff
@@ -117,9 +117,25 @@ http.outcall {
     # (think of a shop system which tries to fetch prices for multiple items), the connect timeout might
     # add up pretty quickly. Therefore, we only try once and then stop trying so that we can render the
     # remaining part of the site (or at least fail gracefully).
-    #
+
+    # The default outcall connect timeout
+    defaultConnectTimeout = 30s
+
+    # The default outcall read timeout
+    defaultReadTimeout = 5m
+
     # Set this value to 0 to disable blacklisting.
     connectTimeoutBlacklistDuration = 10s
+
+    # Provides system wide settings for time-critical requests.
```

Comment:
This is a very coarse and loose description. I'd rather define a proper timeout on the actual outcall itself once created instead of defining yet another global default....